cpu/esp8266/README.md

```diff
@@ -613,22 +636,19 @@ SDCARD_SPI_PARAM_CS    | SPI0_CS0_GPIO | can be overridden
 
 </center>
 
-The GPIO used as CS signal can be overridden by an [application-specific board configuration](#esp8266_application_specific_board_configuration).
+The GPIO used as CS signal can be overridden by [application-specific board configurations](#esp8266_application_specific_board_configuration).
 
 
+\anchor esp8266_app_spec_conf
 # <a name="esp8266_application_specific_configurations"> Application-Specific Configurations </a> &nbsp;[[TOC](#esp8266_toc)]
 
-Configuration used for peripheral devices and for device driver modules, such as GPIO pins, bus interfaces or bus speeds are typically defined in the board-specific configurations ```board.h``` and ```periph_conf.h``` or in the driver parameter configuration ```< driver>_params.h```. Most of these definitions are enclosed by
-```
-#ifndef ANY_PARAMETER
-...
-#endif
-```
-constructs, so it is possible to override them by defining them in front of these constructs.
+The board-specific configuration files ```board.h``` and ```periph_conf.h``` as well well as the driver parameter configuration files ```<driver>_params.h``` define the default configurations for peripherals and device driver modules. These are, for example, the GPIOs used, bus interfaces used or available bus speeds. Because there are many possible configurations and many different application requirements, these default configurations are usually only a compromise between different requirements.
```

Comment:
Extra `well`

Reply:
And I think this line could be split, but this comment would apply to large parts of this file.

Reply:
I remember to out discussion regarding 80 characters per line also for documentation. These files were written before. I you insist on it, I can split the lines.